lib/fluent/command/fluentd.rb

```diff
@@ -115,6 +126,19 @@
   opts[:gem_install_path] = s
 }
 
+op.on('-x', '--signame INTSIGNAME', "an object name which is used for Windows Service signal (Windows only)") {|s|
+  opts[:signame] = s
+}
+
+op.on('--reg-winsvc MODE', "install/uninstall as Windows Service. (i: install, u: uninstall) (Windows only)") {|s|
+  opts[:regwinsvc] = s
+}
+
+op.on('--reg-winsvc-fluentdopt OPTION', "specify fluentd option paramters for Windows Service. (Windows only)") {|s|
+  opts[:fluentdopt] = s
+}
+
```

Comment:
How about showing above windows-only options only when `Fluentd.windows?` is true?